### PR TITLE
Bump version to 2.1.0

### PR DESCRIPTION
**Pull request by somexpert**

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko, @jason-ellis

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,14 @@ Change Log
...
 
+* Added filtering to the groups and permissions selection boxes on the invitation page.
+* Added check for the usernames and emails on the invitation page to show whether they're in use before submitting the form.
 * Added setting to show emails on the accounts page to superusers and users who can create invitations.
 * Added setting to limit the timeframe for showing open invitations and registrations on the accounts page.
-* Made some small UI tweaks on the invitations and accounts pages.
+* Tweaked the UI on the invitations and accounts pages.
 * Fixed remaining sorting difference that occurred after resending invitation email.
+* Fixed some other small UI errors, particularly on the invitations and accounts pages.
+* Changed the dev environment to output emails directly stdout.
```
Comment (ldko) [inline CHANGELOG.md]: Do you want this to say "directly _to_ stdout"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,10 +4,14 @@ Change Log
...
 -----
 
+* Added filtering to the groups and permissions selection boxes on the invitation page.
+* Added check for the usernames and emails on the invitation page to show whether they're in use before submitting the form.
 * Added setting to show emails on the accounts page to superusers and users who can create invitations.
 * Added setting to limit the timeframe for showing open invitations and registrations on the accounts page.
-* Made some small UI tweaks on the invitations and accounts pages.
+* Tweaked the UI on the invitations and accounts pages.
 * Fixed remaining sorting difference that occurred after resending invitation email.
+* Fixed some other small UI errors, particularly on the invitations and accounts pages.
```
Comment (ldko) [inline CHANGELOG.md]: I would find it helpful if the changes described on line 11 and 13 were more specific. If for some reason it doesn't make sense to make them more specific, then I would probably combine the lines.

File: CHANGELOG.md (hunk 1 above)
Comment (somexpert) [reply CHANGELOG.md]: ha, yep, I _did_ want it to say that

File: CHANGELOG.md (hunk 2 above)
Comment (somexpert) [reply CHANGELOG.md]: Alright, I'll add some more specifics here

**Pull request merged**